actions/validate-domain/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: validate-domain
+description: "Validate that helm/Chart.yaml contains annotations.domain with an allowed value"
+
+inputs:
+  chart-file:
+    description: "Path to Chart.yaml"
+    required: false
+    default: "helm/Chart.yaml"
+  allowed-domains:
```

Comment:
This should be a constant here and not as input of the action, since developers might set this as they want and we won't be able to force the domain values